src/main/java/tracko/logic/commands/order/EditOrderCommand.java

```diff
@@ -59,6 +59,11 @@ public class EditOrderCommand extends Command {
     public static final String MESSAGE_ORDER_ALREADY_MARKED =
             "An order marked as paid and/or delivered cannot be edited!";
 
+    public static final String MESSAGE_QUANTITY_INVALID = "Quantity should be an integer less than 2147483648";
```

Comment:
Minor nitpick, but I think it would be easier to read if you separate the numbe with commas

`2,147,483,648`